freetextresponse/freetextresponse.py

```diff
@@ -279,6 +279,17 @@ def student_view(self, context={}):
             (Fragment): The HTML Fragment for this XBlock, which determines the
             general frame of the FreeTextResponse Question.
         """
+        student_id = self.get_student_id()
+        display_other_responses = self.display_other_student_responses
+        student_answer = self.student_answer
+        if student_answer and display_other_responses:
+            other_responses = self.get_other_answers(student_id)
+        else:
+            other_responses = []
+        if other_responses:
```

Comment:
Since this `if` check is purely presentational, shouldn't we move it to the template?

Instead of the backend telling the frontend whether or not to display an element,
we should instead delegate that decision to the template.

So then the template can rationalize,
"if I don't have other responses, I shouldn't try to display them".